client/src/features/post/postComponenet.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { FunctionComponent } from "react";
+import Description from "../../pages/create-post/description";
+import ChooseSong from "../../pages/create-post/choose-song";
+import { FaEdit, FaTrashAlt, FaPlay, FaRegHeart, FaComment, FaReply} from "react-icons/fa";
+import { RiStackFill, RiPlayList2Fill } from "react-icons/ri";
+
+
+export interface SongInfo {
+    name : string
+    albumCoverPath: string
+    artistName: string
+
+}
+
+interface PostComponentProps {
+    username: string
+    profilePath: string
+    authorCaption: string
+    song: SongInfo
+
+
+
+}
+
+/*
+
+    Example PostComponent:
+
+     <PostComponent username="McLovin" profilePath="/2357380.jpg" authorCaption="I never lie, including about my age" song={{
+        albumCoverPath: "/final1.png",
+        name: "The Man Who Never Lied",
+        artistName: "Maroon 5"
+      }}/>
+
+
+
+
+
+*/
+
+
+
+
+
+
+
+//PostComponent Feature
+//TODO: 
+// Figure out how to contrain comment data structure in the props of this component
+// Figure out how to expand PostComponent to show comments
+// Button Handling for all buttons
+
+function PostComponent(prop: PostComponentProps) {
+
+    return (
+        <div className= "flex flex-col h-[346px] w-[600px] bg-[#334155] rounded-lg">
```

Comment:
I would n'tuse this bracket syntax, just so that we are staying within tailwinds styling constraints that will make our site more consistent. Plus, we will want this component to be mobile responsive, but we can deal with that later.